PearlPCSup/devPearl.db

```diff
@@ -147,28 +157,123 @@ record(longin, "$(P)MN_PRESSURE"){
     field(INP, "$(P)STATUS_ARRAY.[11] CP MS")
     field(DTYP, "Soft Channel")
     field(EGU, "bar")
+	info(archive, "VAL")
 }
 
 record(longin, "$(P)PRESSURE:SP:RBV"){
     field(DESC, "Get Setpoint (Nominal) Pressure")
     field(INP, "$(P)STATUS_ARRAY.[12] CP MS")
     field(DTYP, "Soft Channel")
     field(EGU, "bar")
+	info(archive, "VAL")
 }
 
 record(longin, "$(P)MX_PRESSURE"){
     field(DESC, "Get Max Pressure")
     field(INP, "$(P)STATUS_ARRAY.[13] CP MS")
     field(DTYP, "Soft Channel")
     field(EGU, "bar")
+	info(archive, "VAL")
 }
 
 record(longin, "$(P)PRESSURE"){
     field(DESC, "Get Pressure")
     field(INP, "$(P)STATUS_ARRAY.[14] CP MS")
     field(DTYP, "Soft Channel")
-    field(FLNK, "$(P)PRESSURE_COMPARISON")
+    field(FLNK, "$(P)HIGH_PRESSURE_RESET_CHECK")
     field(EGU, "bar")
+	info(archive, "VAL")
+}
+
+record(calc, "$(P)INPUTS:_RAW") {
+    # .A written to from protocol
+    field(INPB, "$(P)STATUS_ARRAY CP MS")
+	field(CALC, "A")
+}
+
+record(mbbiDirect, "$(P)INPUTS") {
+    field(INP, "$(P)INPUTS:_RAW CP MS")
+	# B8 = Emergency Stopped released
```

Comment:
Do you need to use an intermediate calc record - can you not read directly into an mbbiDirect from the record redirection?

Reply:
Ah i see, via the `INPB`. I am not sure what would happen with redirection on disconnection

Reply:
I think it never notices any alarm - which is why we have this pattern elsewhere (sometimes we use it via `error_setter.template`, but in this case I thought it was simple enough not to need that template. I'm happy to use the template if you prefer though.